src/openMVG/matching_image_collection/Cascade_Hashing_Matcher_Regions_AllInMemory.cpp

```diff
@@ -144,7 +144,7 @@ void Match
 #endif
     for (int j = 0; j < (int)indexToCompare.size(); ++j)
     {
-      const size_t J = indexToCompare[j];
+      size_t J = indexToCompare[j];
```

Comment:
Moreover, it seems that in c++11 standard there's no const arguments on std::make_pair : 

http://en.cppreference.com/w/cpp/utility/pair/make_pair